### ISSUE TITLE
Ability to disable, reset or empty rate limit temporarily?

### ISSUE THREAD

**Issue by parisk** [jsocol/django-ratelimit#119]
Hi there,

I am using django-ratelimit in one of my projects and I am running into the following issue:

I am performing functionality tests in various rate-limited endpoints but some times when I run the test suite, django-ratelimit rejects some of the test client's requests with error 429 (I guess it gets lots of requests in a fraction of time).

Disabling it completely is not an option, because I am testing a few endpoints that they are indeed rate limited and behave consistently.

How would you propose me to handle this case?

Thanks a lot!

**Comment by adamchainz** [comment]
```py
with override_settings(RATELIMIT_ENABLE=False):
    do_the_thing()
```

or even the other way round, set `RATELIMIT_ENABLE` to `False` in your base test settings and then override it for the tests that need to test it.

**Comment by parisk** [comment]
That's amazing, thanks!

**Comment by maxmalysh** [comment]
Unit testing a rate-limited view is a common practice. It would be nice to have this documentation.

**Comment by jsocol** [comment]
Fixed in #138! Thanks for the suggestion :) (and @adamchainz for the fix!)
